.github/workflows/test.yml

```
@@ -0,0 +1,88 @@
+      - name: Cache Composer packages
+        id: composer-cache
+        uses: actions/cache@v4
+        with:
+          path: |
+            ~/.cache/composer
+            ~/${{ env.NEOS_FOLDER }}Packages
+          key: php-${{ matrix.php-versions }}-${{ matrix.dependencies }}${{ hashFiles('**/composer.json') }}
+          restore-keys: php-${{ matrix.php-versions }}-${{ matrix.dependencies }}
+
+      - name: Install dependencies
+        run: |
```

actionlint (1.7.8)

property "composer-arguments" is not defined in object type {dependencies: string; neos-versions: number; php-versions: number}

(expression)